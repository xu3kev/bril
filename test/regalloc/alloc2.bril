../reginterp/alloc2.bril