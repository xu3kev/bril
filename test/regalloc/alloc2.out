main {
	vj: int = const 1;
	r_04: int = const 5;
	r_01: int = const 1;
	r_03: int = const 1;
	r_02: int = add vj r_01;
	r_01: int = sub r_04 r_03;
	r_02: int = mul r_02 r_01;
	r_01: int = const 2;
	r_03: int = add vj r_01;
	r_01: int = const 3;
	r_04: int = add vj r_01;
	vb: int = id r_02;
	r_02: int = const 5;
	r_01: int = add r_03 r_02;
	r_01: int = id r_01;
	r_04: int = add r_04 r_02;
	vj: int = id vb;
	print vj;
	print r_04;
	print r_01;
}
